roles/sno_installer/tasks/10_get_oc.yml

```
@@ -101,7 +101,7 @@
   vars:
     ocp_stable_url: https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp
   ansible.builtin.unarchive:
-    src: "{{ ocp_stable_url }}/stable/openshift-client-linux.tar.gz"
+    src: "{{ ocp_stable_url }}//stable/openshift-client-linux-amd64-rhel{{ ansible_distribution_major_version }}.tar.gz"
```

extra `/` is causing download issues in the last job :P
```suggestion
    src: "{{ ocp_stable_url }}/stable/openshift-client-linux-amd64-rhel{{ ansible_distribution_major_version }}.tar.gz"
```

---

my bad, fixed it